Flow.Launcher/ViewModel/ResultsViewModel.cs

```diff
@@ -290,7 +290,7 @@ public void Update(List<ResultViewModel> newItems, CancellationToken token = def
                 if (Count == 0 && newItems.Count == 0 || _token.IsCancellationRequested)
                     return;
 
-                if (editTime < 10 || newItems.Count < 30)
```

Comment:
editTime is no longer used, remove it?

Reply:
This change may resolve the issue. The issue looks like to be the time consuming when adding elements separately.

Reply:
> editTime is no longer used, remove it?

yeah remove it

Reply:
why you went from newItems.Count < 30 to newItems.Count < 100? What's the reason for changing this number